submissions/management/commands/analyze_uploaded_file_sizes.py

```diff
@@ -0,0 +1,157 @@
+"""
+Management command to analyze recently-uploaded files and create a report containing:
+
+Per course:
+    1) total bytes of files uploaded
+    2) count of users who have uploaded a file
+    3) rough bytes-per-user, #1 divided by #2
+
+This report is printed to stdout with a header and values as a tabbed stream.
+
+Arguments:
+    -start_date (optional): The most recent day to include in the report. Defaults to today.
+    -end_date (optional): The longest-ago day to include in the report.
+                          Defaults to <today - 30 days>.
+                          The maximum value for this argument is 2019-11-01.
+                          Before this day, file size data is unavailable.
+
+    both arguments must be in the form yyyy-mm-dd
+"""
+
+from __future__ import absolute_import
+
+import datetime
+import json
+
+from django.core.management.base import BaseCommand, CommandError
+
+from submissions.models import Submission
+
+DATE_FORMAT = '%Y-%m-%d'
+LAST_DATE = datetime.date(2019, 11, 1)
+HEADER = u'Course ID\tUsers with Uploaded Files\tTotal Uploaded Bytes\tAverage Upload per User'
+
+
+class Command(BaseCommand):
+    """
+    Example usage:
+
+        ./manage.py lms analyze_uploaded_file_sizes --start_date=2020-01-30 --end_date=2019-12-31
+
+    """
+    help = u'Collects and prints stats about ORA file upload usage'
+
+    def add_arguments(self, parser):
+        """
+        Add arguments to the command parser.
+        """
+        today = datetime.date.today()
+        thirty_days = datetime.timedelta(days=30)
+        thirty_days_ago = today - thirty_days
+
+        def parse_date(s):
+            datetime.datetime.strptime(s, DATE_FORMAT).date()
+
+        parser.add_argument(
+            '--start_date',
+            type=parse_date,
+            default=today,
+            help=u"The most recent day to include in the report. Defaults to today."
+        )
+        parser.add_argument(
+            '--end_date',
+            type=parse_date,
+            default=thirty_days_ago,
+            help=(u"The longest-ago day to include in the report. Defaults to <today - 30 days>. "
```

Comment:
This is the opposite of what I would expect `end-date` to mean. :) . Should we change to `min-date` and `max-date` instead?